### ISSUE TITLE
Accordion header '+' image missing alt attribute

### ISSUE THREAD

**Issue by adritek** [openSUSE/mentoring#213]
### Accessibility/Responsive Issue
**Page**  
[Home/landingpage](https://www.uyuni-project.org/)

**Description**  
Alt text is missing on the image tags for the '+' svgs on the accordion section headers

**Proposed Solution**  
This is an interesting one, because there's a couple ways to think about it...
The accordion is already open and displays the description under the title to the user - is the '+' an icon as in, does it convey meaning, or is it decorative. 
A '+' would universally be understood as addition, but in the context of a collapsible accordion maybe a 'v' is more informative for 'expand' and '^' for 'collapse would be a better choice.
Or are we trying to build semantic markup.

**Solutions**
Solution 1 (quick): Give the <img> in the accordions an alt attribute.
Solution 2 (semantic): Pull the <svg> tag out of the image, and use aria-hidden (if decorative) and role attributes.

**Additional Notes**  
- [Article](https://docs.fontawesome.com/web/dig-deeper/accessibility) on font icons/icons and svg for consideration.
- [Article](https://css-tricks.com/accessible-svgs/#aa-conclusion) on semantics.

**Screenshot**  
![Screen Shot 2024-05-09 at 1 22 43 am](https://github.com/openSUSE/mentoring/assets/997506/c48a1c58-2be8-42af-92aa-b1574b93ce24)

**Comment by ddemaio** [comment]
Hi @adritek GSoC already selected the projects listed. These all were added after the selections were made. Is there something specific you are using all these issues you've added for? Should I close them?

**Comment by adritek** [comment]
Thanks for responding @ddemaio - 
The TLDR is: I made a bunch of WCAG accessibility issues under the wrong repo, and they need to be transferred to the right repo. 

The issues I raised were all incorrectly made under [mentoring](https://github.com/openSUSE/mentoring/issues), and I should have put them under [Uyuni project](https://github.com/uyuni-project/uyuni-project.github.io/issues).
I cannot transfer them to the appropriate project (understandably), are you able to do this? And apologies.

**Comment by deneb-alpha** [comment]
moved to https://github.com/uyuni-project/uyuni-project.github.io/issues/111
